datahub-web-react/src/graphql/dataset.graphql

```diff
@@ -1,3 +1,38 @@
+fragment simplifiedGlossaryTerms on GlossaryTerms {
+    terms {
+        term {
+            urn
+            name
+            type
+            hierarchicalName
+            properties {
+                name
+                description
+                definition
+                termSource
+                customProperties {
+                    key
+                    value
+                }
+            }
+            ownership {
+                ...ownershipFields
+            }
+            parentNodes {
+                count
+                nodes {
+                    urn
+                    type
+                    properties {
+                        name
+                    }
+                }
+            }
```

Comment:
the change here is amazing (so that we don't also fetch `children` of nodes in the `parentNodesFields` when we don't actually need to). in fact, so good that I think we should apply this everywhere!

In order to do that, I think you can actually drop this change and simply change the fragment `parentNodesFields` from:
```
fragment parentNodesFields on ParentNodesResult {
    count
    nodes {
        ...glossaryNode
    }
}

```
to:
```
fragment parentNodesFields on ParentNodesResult {
    count
    nodes {
        urn
        type
        properties {
            name
        }
    }
}
```

Reply:
then this performance change will benefit all entities and wherever we fetch parentNodes (on existing nodes as well)

Reply:
This is awesome!

Reply:
okay! i'm going to merge this PR once CI passes and then go and make the additional change right when it gets in, just cuz it'll be a nice simple fix.

Thanks again for putting this up!